tests/playwright/specs/specialrecentchanges.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable mediawiki/valid-package-file-require */
+/* eslint-disable one-var */
+'use strict';
+const Api = require( '../helpers/Api' );
+const querystring = require( 'querystring' );
+
+describe( 'Special:RecentChanges', function () {
+	it( 'shows page creation', async () => {
+		const content = 'content-Iñtërnâtiônàlizætiøn';
+		const name = 'Name-Iñtërnâtiônàlizætiøn';
+		const bot = await Api.bot();
+
+		const changesList = '.mw-changeslist';
+		const changeListTitles = '.mw-changeslist-title';
+
+		await bot.edit( name, content );
+		await page.goto(
+			`${global.baseUrl}/index.php?${querystring.stringify( 'Special:RecentChanges' )}# `
```

Comment:
I think this should just work:

    await page.goto(`${global.baseUrl}/wiki/Special:RecentChanges`)

Reply:
Ok, try this. 😄 

    await page.goto(`${global.baseUrl}/index.php?title=Special:RecentChanges`)

Reply:
This has worked. Thanks